app/[locale]/institute/sections/sports/page.tsx

```diff
@@ -9,9 +19,336 @@ export default async function Sports({
 }) {
   const text = (await getTranslations(locale)).Section.Sports;
 
-  const section = (await db.query.sections.findFirst({
-    where: (section, { eq }) => eq(section.urlName, 'sports'),
-  }))!;
```

Comment:
Um, why are we removing this fetch? We need to render useful attributes in there like `email`, `aboutUs`, and `headFaculty`.

Reply:
But they aren't locale specific are they ?

Reply:
That can be said for everything in the DB.

Translations for now are only done for static data. The plan for data from the DB is that we run a microservice that interjects and translates before sending the output. But until we do that, we won't be translating anything from the DB.